tests/app/delivery/test_send_to_providers.py

```diff
@@ -450,21 +450,27 @@ def test_send_email_should_use_service_reply_to_email(
     )
 
 
-def test_get_html_email_renderer_should_return_for_normal_service(sample_service):
-    options = send_to_providers.get_html_email_options(sample_service)
+def test_get_html_email_renderer_should_return_for_normal_service(sample_notification_model_with_organization):
+    options = send_to_providers.get_html_email_options(
+        sample_notification_model_with_organization,
+        app.govdelivery_client)
```

Comment:
Please use a mock provider to not tie tests to any of the real providers.

Reply:
Done